src/main/kotlin/utils/CoverageFileStatistics.kt

```diff
@@ -0,0 +1,64 @@
+package utils
+import org.xml.sax.Attributes
+import org.xml.sax.InputSource
+import org.xml.sax.helpers.DefaultHandler
+import java.io.ByteArrayInputStream
+import java.io.InputStream
+import javax.xml.parsers.SAXParserFactory
+import kotlin.math.min
+
+object CoverageFileStatistics {
+
+    private class Handler() : DefaultHandler() {
+
+        var currentPrefix = ""
+        var coveredMethods = 0
```

Comment:
when reading the class top to bottom, I thought the purpose of this was to aggregate the total number of covered methods. turns out, it's a cache that is overwritten each time a coverage counter is read. Please document this here. Same for the next variable